k8s/config/oss/helm/kustomization.yaml

```diff
@@ -6,8 +6,6 @@
 bases:
 - "../default"
 patches:
-- path: "./manager-patch.yaml"
-- path: "./namepsace-patch.yaml"
```

Comment:
The path seems to be ./manager/namespace-pathc.yaml? Why is it working before?

Reply:
There used to be a typo in both reference and yaml name. Has fixed them.